libnavigation-ui/build.gradle

```diff
@@ -76,6 +76,8 @@ dependencies {
 
   // Mapbox Maps SDK
   api dependenciesList.mapboxMapSdk
+  // workaround for a missing transitive artifact
+  implementation "com.mapbox.mapboxsdk:mapbox-android-sdk-gl-core:1.8.1"
```

Comment:
Do we have a plan to remove this?

Reply:
I simply added back what I saw was there before. I don't know the answer to your question.

Reply:
cc @LukasPaczos

Reply:
Also asking because the comment is "// workaround for a missing transitive artifact"

Does that mean it's temporary? or should it fit in as something like dependenciesList.mapboxGLCore

Reply:
This should be resolved, we can remove and retest.

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/3312